trpcultivate_phenotypes/tests/src/Kernel/PluginValidatorTest.php

```diff
@@ -378,6 +387,63 @@ public function testDataFilePluginValidator() {
     $this->assertEquals($validation[ $scope ]['status'], $status);
   }
 
+  /**
+   * Test Headers Plugin Validator.
+   */
+  public function testScopePluginValidator() {
```

Comment:
```suggestion
  public function testColumnHeaderPluginValidator() {
```

Reply:
good catch

Reply:
This one doesn't seem to have been applied.